File: src/main/java/com/jakimenko/rainforecastbot/controlers/TestController.java
package com.jakimenko.rainforecastbot.controlers;

import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RestController;

@RestController
public class TestController {

    @GetMapping
    public BaseResponse testMethod() {
        return new BaseResponse(200, "main test method");
    }
}

File: src/main/kotlin/com/jakimenko/rainforecastbot/controllers/TestController.kt
package com.jakimenko.rainforecastbot.controllers

import org.springframework.web.bind.annotation.*

@RestController
@RequestMapping("/")
class TestController {

    @GetMapping("test")
    fun testMethod() = "test method"
        // System.getenv("TGTOKEN")

    @GetMapping
    fun rootTestMethod() = "Root test method"

    // http://localhost:8080/double?msg=mytext
    @GetMapping("double")
    fun doubbledMsg(@RequestParam("msg") msg: String) = msg+msg
}
